src/test/java/net/snowflake/client/jdbc/SnowflakeDriverLatestIT.java

```diff
@@ -1766,6 +1766,93 @@ public void testS3PutInGS() throws Throwable {
     }
   }
 
+  @Test
+  @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
+  public void testGcpAzureReEncryption() throws Throwable {
```

Comment:
How this test is connected to reencryption? What is a difference comparing to other PUT/GET tests? Maybe worth commenting?

Reply:
Yes, good idea. Let me add a better description.